### PR TITLE
pf4: Use SVG spinner to fix direction and glitches

### PR DESCRIPTION
**Pull request by garrett**
Blocked on:
* [x] #174 

Addresses https://github.com/cockpit-project/cockpit/issues/15573

### CODE REVIEW

**Comment by garrett** [comment]
This doesn't seem to work, but it also doesn't seem to break things.

My guess is that the version of PatternFly we're using for cockpit-ostree is too old. Once that's upgraded, this PR should work.

**Comment by KKoukiou** [comment]
@garrett this needs that you change the PF version in the package.json

**Comment by garrett** [comment]
package.json uses the same version as Cockpit, however, the spinners are still going the wrong way and I can't find any additional use of a React `<Spinner>` in the code.

**Comment by KKoukiou** [comment]
> package.json uses the same version as Cockpit, however, the spinners are still going the wrong way and I can't find any additional use of a React `<Spinner>` in the code.

We are interested in @patternfly/react-core and it's not the same version as in Cockpit. 
https://github.com/cockpit-project/cockpit-ostree/blob/master/package.json#L53

**Comment by KKoukiou** [comment]
Oh I know, there was a breaking change release of course.

**Comment by martinpitt** [comment]
> I am not sure why the npm-update script does not want to update patternfly here.

It only updates minor versions, not major ones.

**Comment by martinpitt** [comment]
Unblocked, please rebase.

**Comment by KKoukiou** [comment]
> Unblocked, please rebase.

It's actually fine, there are not conflicts here. I will just test rebase locally and test it.

File: src/app.jsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ const OriginSelector = ({ os, remotes, branches, branchLoadError, currentRemote,
                                 id="check-for-updates-btn"
                                 isDisabled={!!client.local_running || !!progressMsg}
                                 onClick={checkForUpgrades}>
-                            {!(!!client.local_running || !!progressMsg) ? _("Check for Updates") : <Spinner size="sm" />}
+                            {!(!!client.local_running || !!progressMsg) ? _("Check for Updates") : <Spinner isSVG size="sm" />}
```
Comment (KKoukiou) [inline src/app.jsx:193]: This should be changed to use isLoading property.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**